test/unit/mdc-list/mdc-list.test.js

```diff
@@ -226,11 +225,11 @@ test('adapter#setTabIndexForListItemChildren sets the child button/a/radio/check
   document.body.appendChild(root);
   const listItems = root.querySelectorAll('.mdc-list-item');
 
-  for (let index = 0; index < listItems.length; index++) {
-    assert.equal(0, listItems[index].querySelectorAll('[tabindex="0"]').length);
-    component.getDefaultFoundation().adapter_.setTabIndexForListItemChildren(index, 0);
-    assert.equal(1, listItems[index].querySelectorAll('[tabindex="0"]').length);
-  }
+  component.getDefaultFoundation().adapter_.setTabIndexForListItemChildren(0, 0);
```

Comment:
The descriptions of this test and in the README need updating for the new behavior.